src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Initialization.cs

```diff
@@ -42,13 +71,204 @@ static CryptoInitializer()
                 throw new InvalidOperationException();
             }
         }
+#pragma warning restore CA1810
 
         internal static void Initialize()
         {
             // No-op that exists to provide a hook for other static constructors.
         }
 
+        private static bool GetMemoryDebug(string name)
+        {
+            string? value = Environment.GetEnvironmentVariable(name);
+            if (int.TryParse(value, CultureInfo.InvariantCulture, out int enabled))
+            {
+                if (enabled == 2)
+                {
+                    EnableTracking();
+                }
+                else if (enabled == 3)
+                {
+                    GetIncrementalAllocations();
+                }
+
+                return enabled == 1;
+            }
```

Comment:
I would like these magic constants documented at the top of the file.

Also, the logic seems weird to me, the function returns true only if `enabled == 1`, so using values 2 or 3 will not lead to setting custom memory alloc routines in OpenSSL?